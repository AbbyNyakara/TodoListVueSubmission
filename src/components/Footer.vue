<template>
  <div class="footer">
    <small>Drag and drop to reorder list</small>
  </div>
</template>

<script>
  export default {
    name: "Footer"
  }

</script>

<style scoped>
  .footer {
    display: grid;
    place-items: center;
    margin-top: 2rem;
    color: hsl(234, 39%, 85%);
  }

  small {
    text-align: center;
  }
</style>