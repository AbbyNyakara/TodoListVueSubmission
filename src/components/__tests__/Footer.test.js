import { describe, expect, it } from 'vitest';
import { mount } from '@vue/test-utils';
import Footer from '../../components/Footer.vue';

describe("My Footer component", () => {
  it("Renders footer component", ()=> {
    const wrapper = mount(Footer);
    expect(wrapper.html()).toMatchSnapshot()
  })
})