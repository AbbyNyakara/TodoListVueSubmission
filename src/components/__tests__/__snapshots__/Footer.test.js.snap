// Vitest Snapshot v1

exports[`My Footer component > Renders footer component 1`] = `"<div class=\\"footer\\" data-v-4896eafd=\\"\\"><small data-v-4896eafd=\\"\\">Drag and drop to reorder list</small></div>"`;
